libplatsupport/CMakeLists.txt

```diff
@@ -134,7 +134,7 @@ if(KernelPlatImx8mq OR KernelPlatformImx8mm-evk)
     target_include_directories(platsupport PUBLIC plat_include/imx8m)
 endif()
 
-if(KernelPlatformZynqmpZcu102 OR KernelPlatformZynqmpUltra96 OR KernelPlatformZynqmpUltra96v2)
+if(KernelPlatformZynqmp)
     # This still needs to be cleaned up, so there is a mach folder "zynymq" and
```

Comment:
I know this isn't part of the PR, but "zynqmp" is mangled here. Can you fix?

Reply:
I've added a commit that fixed the comment